www/js/jquery.blink.js

```diff
@@ -0,0 +1,16 @@
+(function ($) {
+	$.fn.blink = function (options) {
+		var defaults = { delay: 500 };
+		var options = $.extend(defaults, options);
+		return $(this).each(function (idx, itm) {
+			setInterval(function () {
+				if ($(itm).css("visibility") === "visible") {
+					$(itm).css('visibility', 'hidden');
+				}
+				else {
+					$(itm).css('visibility', 'visible');
+				}
+			}, options.delay);
+		});
+	}
+} (jQuery))
```

Comment:
This jQuery plugin seems a bit silly for something as simple as blinking text. Why not use CSS3 animations?